roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -161,6 +161,13 @@
     - cloud-provider
     - facts
 
+- name: Show present IPv6 addresses
+  debug:
+    msg: |
+      ansible_all_ipv6_addresses: {{ ansible_all_ipv6_addresses }}
+      ansible_default_ipv6: {{ ansible_default_ipv6 }}
+    verbosity: 0
```

Comment:
Why do we need such debug task ? You can print the IPs in the `msg` of the assert instead of always printing

Reply:
Vars added to output of assert instead, as suggested